engine/apps/mobile_app/urls.py

```diff
@@ -25,3 +33,12 @@
 urlpatterns += [
     optional_slash_path("fcm_relay", FCMRelayView.as_view(), name="fcm_relay"),
 ]
+
+if settings.MOBILE_APP_GATEWAY_ENABLED:
+    urlpatterns += [
+        re_path(
+            r"^gateway/(?P<downstream_backend>\w*)/(?P<downstream_path>.*)$",
```

Comment:
basically this would proxy the following request:
```
GET /mobile_app/v1/gateway/incident/test/123/someotherthing/456
```
into:

```python3
downstream_backend = "incident"
downstream_path = "test/123/someotherthing/456"
```